Assets/PatchKit Patcher/Scripts/AppUpdater/Commands/InstallContentCommand.cs

```diff
@@ -159,18 +170,16 @@ private IUnarchiver CreateUnrachiver(string destinationDir, out string usedSuffi
             }
         }
 
-        private void InstallFile(string fileName, string packageDirPath, string suffix, CancellationToken cancellationToken)
+        private void InstallFile(SourceFile sourceFile, CancellationToken cancellationToken)
         {
-            DebugLogger.Log(string.Format("Installing file {0}", fileName+suffix));
-
-            string sourceFilePath = Path.Combine(packageDirPath, fileName+suffix);
+            DebugLogger.Log(string.Format("Installing file {0}", sourceFile.Name));
 
-            if (!File.Exists(sourceFilePath))
+            if (!sourceFile.Exists())
             {
-                throw new InstallerException(string.Format("Cannot find file {0} in content package.", fileName));
+                throw new InstallerException(string.Format("Cannot find file {0} in content package.", sourceFile.Exists()));
```

Comment:
Shouldn't it be `sourceFile.Name` instead of `sourceFile.Exists()` in the `string.Format` here?